src/Templates/InterfacesTemplate.cshtml

```diff
@@ -0,0 +1,81 @@
+@using AutoRest.Core.Utilities
+@using AutoRest.Go
+@using AutoRest.Go.Model
+@using AutoRest.Go.Templates
+@using System;
+@using System.Collections.Generic;
+@using System.Linq;
+
+@inherits AutoRest.Core.Template<AutoRest.Go.Model.CodeModelGo>
+
+@{
+    var content = new Dictionary<string, IEnumerable<MethodGo>>();
+    var imports = new HashSet<string> { PrimaryTypeGo.GetImportLine(package: "context") };
+
+    if (Model.ClientMethods.Any())
```

Comment:
is this if really necessary ? if there are not client methods, I guess it just wont add anything ?

Reply:
I think you're correct, will remove.